packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,87 @@ section: get-started
 import './get-started.css';
 import { Divider } from '@patternfly/react-core';
 
+## PatternFly 5.2 
+
+PatternFly 5.2 is now live! Here are the highlights from this release. 
+
+### Major release preparation
+
+The next major version of PatternFly (v6) doesn't come out until later this year, but our team has started preparing some of its most significant updates.
+
+Our major development work from the past quarter lays the groundwork for our ***upcoming v6 alpha release***, which is slotted for Q1 2024.
+
+#### Design token support 
```

Comment:
The same notes is probably also true here. The release itself is not including any of the penta work. It's true that we've been working hard on it. Maybe rather than devoting a section to design token support here, we could add a 'looking forward' section to the bottom where we call out that the team has been hard at work supporting design tokens and building out PF's theming capabilities? Maybe even tell them to stay tuned for an alpha to be released in the near future?

Reply:
thanks - I think this was a great idea, so that we can still point out where a lot of work time went for this release!